samples/network/manage-ip-address/Program.cs

```diff
@@ -66,65 +62,59 @@ public static async Task RunSample(TokenCredential credential)
                     }
                 };
 
-                publicIPAddress = (await publicIPAddresses.StartCreateOrUpdate(rgName, publicIPAddressName1, publicIPAddress)
-                     .WaitForCompletionAsync()).Value;
-
-                Utilities.Log("Created a public IP address");
-                // Print public IP address details
-                Utilities.PrintIPAddress(publicIPAddress);
+                var publicIPAddressContainer = resourceGroup.GetPublicIPAddresses();
```

Comment:
not force to always use 'var' or strong type, but should be consistent in one file/project.

Reply:
For return values of some operations, there is an implicit conversion from `Response<Type>` to `Type`. For those cases, we need to add explicitly type declaration here. For this sample, we use strong type for all those case, and `var` for the rest, which I hope will make it more clear. I'll add some comments to emphasize that.